config.js.example

```diff
@@ -16,22 +16,35 @@ exports.ConfigSettings = {
             routingServers: [{
                 name: "sharda",
                 serverIP: "127.0.0.1",
-                serverPort: 7780
+                serverPort: 7780,
+                hidden: false
             },
             {
                 name: "shardb",
                 serverIP: "127.0.0.1",
-                serverPort: 7781
+                serverPort: 7781,
+                hidden: false
             }]
         },
         // This server listens on another port, and is not load balanced with the others
         // You can switch to it at any time using `/oshard` or by joining 7778
+        // Hidden servers will not pop up in server list but players still can access
+		// using the server name as command.
+		// Example: `hshard` will not visible on server list but players can access
+		// by typing using `/hshard`
```

Comment:
Indentation seems wrong here

Reply:
Fixed, because i'm using TAB button instead of spaces, this thing adds extra TAB space on it, fixed at 42ef0d94fd4fbca798297cc0a03cec56cfc71af6